extensions/DynamicPageList/DPLMain.php

```diff
@@ -3074,8 +3074,8 @@ private static function getSubcategories($cat,$sPageTable,$depth) {
         $dbr =& wfGetDB( DB_SLAVE );
         $cats=$cat;
         $res = $dbr->query("SELECT DISTINCT page_title FROM ".$dbr->tableName('page')." INNER JOIN "
-                .$dbr->tableName('categorylinks')." AS cl0 ON ".$sPageTable.".page_id = cl0.cl_from AND cl0.cl_to='"
-               .str_replace(' ','_',$cat)."'"." WHERE page_namespace='14'");
+                .$dbr->tableName('categorylinks')." AS cl0 ON ".$sPageTable.".page_id = cl0.cl_from AND cl0.cl_to="
+               . $dbr->addQuotes(str_replace(' ','_',$cat))." WHERE page_namespace=".NS_CATEGORY); # Wikia change - PLATFORM-2422
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)